app/styles/tasks.css

```diff
@@ -6,6 +6,23 @@
   --scrollbar-thumb-hover: #555;
 }
 
+.task-on-time {
+  --primary-text-color: #041187;
```

Comment:
use variables for colors to make it consistent throughout

Reply:
this is a variable that I declared.

Reply:
> use variables for colors to make it consistent throughout

My bad